scripts/test_python_version.py

```diff
@@ -0,0 +1,23 @@
+#!/bin/env python
+
+import platform
+import sys
+
+
+def main():
+    major_v, minor_v, patch_v = platform.python_version_tuple()
+
+    print(f"Found python version: {major_v}.{minor_v}.{patch_v}")
+
+    exit_code = (
+        1 if int(major_v) < 2 or (int(major_v) == 3 and int(minor_v) < 10) else 0
```

Comment:
the source of truth for the python version that we use in the repo is at `requirements/PYTHON_VERSION`. 

It's important that this info in in sync in all parts. Sometimes we do not want to reach this file (e.g. in the Dockefile) so alternatively we add a test in `tests/environment-setup/test_used_python.py` that ensure that these parts are in sync

Reply:
Ok, sorry, i saw the tests you referred to in the chat, but didn't realize there was a clean file. I will check based on that file content.